app/2.0.x/admin-api.md

```diff
@@ -1964,32 +1964,32 @@ HTTP 201 Created
 
 ##### List Plugins Associated to a Specific Route
 
-<div class="endpoint get indent">/routes/{route id}/plugins</div>
+<div class="endpoint get indent">/routes/{route name or id}/plugins</div>
```

Comment:
I have tried adding these changes to the templates we use to generate the docs but it is harder than anticipated. Leaving this PR open as a reminder that this needs to happen.